src/features/Application/ManageApplicationContainer.tsx

```diff
@@ -52,6 +53,7 @@ import { ResetBtn } from '../../shared/Form/ResetBtn';
 import WithToasterContainer from '../../shared/HOC/withToaster';
 // import Sidebar from '../Sidebar/Sidebar';
 
+import Settings from '../../api/settings';
```

Comment:
nit: should be part of a higher import group, with line 44 I think, not with svgs

Reply:
Done